client/src/globals/UserContext.js

```diff
@@ -8,6 +8,22 @@ export const UserContext = createContext({
 export const UserContextProvider = (props) => {
     const [userId, setUserId] = useState("");
     const value = { userId, setUserId };
+    useEffect(() => {
+        let status;
+        fetch("/getcookie", {
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json"
+            }
+        })
+            .then(res => {
+                if (status === 200 && res.success === 'true') return setUserId(res.userId);
```

Comment:
As you have it defined in line 12, the value of `status` is `undefined`. Maybe you meant to access a `status` property of the `res` object?